my-app/.env

```diff
@@ -0,0 +1,7 @@
+
+REACT_APP_API_KEY="AIzaSyB1E1wbhFHmo53B3Gl9IcoapWRzxWw2rtc",
```

Comment:
please don't commit this file :)
I saw that you added `.env` to .gitignore - but you shouldn't commit it :)
Let's handle it on todays call